rossum/password.py

```diff
@@ -0,0 +1,25 @@
+import click
+
+from rossum.lib.api_client import RossumClient
+
+
+@click.group("password")
+def cli() -> None:
+    pass
+
+
+@cli.command(name="change", help="Change password.")
+@click.password_option(required=True)  # type: ignore
+@click.pass_context
+def change_command(ctx: click.Context, password: str) -> None:
+    """
+    Change password for the specific user.
+    """
+    if password:
+        with RossumClient(context=ctx.obj) as rossum:
+            result = rossum.change_user_password(password)
+            click.echo(result.get("detail"))
+            if result.get("detail") == "New password has been saved.":
+                return click.echo(
```

Comment:
Other `rossum` commands do not return anything. They only echo. Consider following the syntax of the already existing commands.